CovidShield/mobile
# Make local development work (in iOS)

## Overview

This PR does a few things in support of being able to run the entire system locally, especially between the mobile app and server:

 - fixes a typo that prevented the diagnosis keys from being retrieved over the RN bridge
 - updates the exposure configuration URL to match the current server version
 - adds a **Run Exposure Check** item to the test menu when the mock server is disabled
 - adds a `forceRefresh` parameter to the exposure notification service to allow the above item to work
 - updates the `.env` to default to localhost for development

Thank you for pairing with me @henrytao-me! 🙏 

## Testing

Ensure that you have the [server](https://github.com/CovidShield/server) repo checked out and running via e.g. `docker-compose`.

1. Launch the app (I tested in iOS).
1. Open the test menu
1. Disable the mock server
1. Press the "Run Exposure Check" item:
![image](https://user-images.githubusercontent.com/934893/82910471-ab59de80-9f38-11ea-8855-340241275aa3.png)
1. Observe in the `server` logs that the exposure configuration was retrieved successfully but the diagnosis keys are not because we try to fetch for the current period.

cc @gavrix @jordanAtShopify
